string type
geometry_msgs/Vector3 color
Pixel top_left
Pixel bottom_right
Pixel[] pixels
